keyboards/matchstickworks/southpad/readme.md

```diff
@@ -0,0 +1,23 @@
+# SouthPad
+
+![Southpad](https://i.imgur.com/Wl71thn.jpg)
```

Comment:
```suggestion
![Southpad](https://i.imgur.com/Wl71thnh.jpg)
```

Reply:
Presuming the appended "h" limits the size of the image? TIL!